### ISSUE TITLE
System.AccessViolationException when scanning any extracted file

### ISSUE THREAD

**Issue by UberNooga** [rickomax/psxprev#40]
Using the latest release, I can't use this program at all as it crashes with the following error any time I try to scan anything. Tried debug mode and it doesn't show the error message but still crashes. Also tried running as admin and in compatibility mode to no avail.

I'm on Windows 10 and my .NET framwork is 4.8.4084.0 if that helps.

![PSXPrevError](https://user-images.githubusercontent.com/22250603/141054278-af674647-c037-469b-9116-d0ad435718db.png)

**Comment by trigger-segfault** [comment]
It's possible this was being caused by the scene control in the preview window not being created on the main thread. In the next upcoming release, the preview window will always be created on the main thread which should *hopefully* solve this. But I'm not entirely sure since I couldn't find any other people having similar issues with OpenTK.

I'll write another comment when the release is up.

**Comment by trigger-segfault** [comment]
The latest release with the changes is up.

However, based on issue #170, have you tried running PSXPrev since restarting your computer?

**Comment by UberNooga** [comment]
Alright, so I've tried again on the new version and now I can't even get the scanner window open (have restarted pc). I get this error:
![image](https://github.com/rickomax/psxprev/assets/22250603/7cd627a6-9f24-4be9-9fcb-b6f97bbadcc2)

Get the same thing when using command line too.

**Comment by trigger-segfault** [comment]
Alright, thanks for checking in again. The scanner window doesn't open now, because the scanner window was changed to open after the preview window (which makes it easier to perform multiple scans). So that's expected.

I'm wondering if SDL2 shouldn't be the one trying to create the graphics context. I've attached a build with two different exe versions that try to solve that by telling the program how to setup the graphics context. Hopefully this solves the issue. 🤞 
* `PSXPrev_PreferNative.exe` - Try this first
* `PSXPrev_PreferX11.exe`

[PSXPrev_TestBackends.zip](https://github.com/rickomax/psxprev/files/12728753/PSXPrev_TestBackends.zip)
